scheduler-server/scheduling-dsl-compiler/src/libs/scheduler-edsl-fluent-api.ts

```diff
@@ -1,8 +1,11 @@
 import * as AST from './scheduler-ast.js';
+import { ActivityType } from "./mission-model-generated-code";
+import {CardinalityGoalArguments} from "./scheduler-ast.js";
```

Comment:
I think `ActivityType` is unused, so we can delete that import.

The `CardinalityGoalArguments` import can also be deleted - instead, we should access it via `AST.CardinalityGoalArguments`.

```suggestion
```